lib/src/controls.dart

```diff
@@ -0,0 +1,309 @@
+import 'dart:async';
+
+import 'package:audio_video_progress_bar/audio_video_progress_bar.dart';
+import 'package:flutter/material.dart';
+
+import '../dart_vlc.dart';
```

Comment:
It is a bit of convention that I'm following, the files inside `src` don't address the outside file directly.

May you change this to `import 'package:dart_vlc/src/the_files_you_need.dart';` like import.